cypress/e2e/po/components/workloads/pod.po.ts

```diff
@@ -19,4 +20,10 @@ export default class PodPo extends CreateEditViewPo {
     // No console request for this one
     terminal.executeMultilineCommand(podYaml, wait);
   }
+
+  openPodShell() {
```

Comment:
This should be in the PO for the pods list (`WorkloadsPodsListPagePo`) rather than the PO for the create/edit view